spec/features/upload_download_mous_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'features/support/not_signed_in'
+require 'features/support/sign_up_helpers'
+
+describe 'the upload and download of MOUs', focus: true do
```

Comment:
I know this is WIP, but want to drop this `focus: true` before merging.

Reply:
Noticed and removed 👍